texar/torch/modules/networks/conv_networks.py

```diff
@@ -349,11 +390,17 @@ def _build_conv1d_hparams(self, in_channels, pool_hparams):
                                    'kernel_size', nconv)
             kernel_size = [_to_list(ks) for ks in kernel_size]
 
-        other_kwargs = self._hparams.other_conv_kwargs or {}
+        other_kwargs = self._hparams.other_conv_kwargs
         if isinstance(other_kwargs, HParams):
             other_kwargs = other_kwargs.todict()
-        if not isinstance(other_kwargs, dict):
-            raise ValueError("hparams['other_conv_kwargs'] must be a dict.")
+            other_kwargs = _to_list(other_kwargs, "other_conv_kwargs", nconv)
+        elif isinstance(other_kwargs, (list, tuple)):
+            if len(other_kwargs) != nconv:
+                raise ValueError("len(hparams['other_conv_kwargs']) must be "
```

Comment:
`len(hparams['other_conv_kwargs'])` looks a bit complicated. Use sth like: The length of the `hparams['other_conv_kwargs']` list

Reply:
Updated in d2c2634